azkaban-web-server/src/web/js/azkaban/view/schedule-sla.js

```diff
@@ -189,6 +189,14 @@ azkaban.ChangeSlaView = Backbone.View.extend({
       var duration = rFlowRule.cells[2].firstChild.value;
       var email = rFlowRule.cells[3].firstChild.checked;
       var kill = rFlowRule.cells[4].firstChild.checked;
+      if (email === false && kill === false) {
+        alert("Unable to create SLA Action. Please set either Kill or Email"
```

Comment:
These should be Bootstrap modal dialogs instead of `alert()` functions to match Azkaban's styles but maybe it can be done separately.

Reply:
Can you provide an example where we use them?

Reply:
As discussed offline, this is being deferred.